### DISCUSSION TITLE
Couple of questions

### DISCUSSION THREAD

**Discussion by bbevren** [CoderLine/alphaTab#479] [Q&A] [answered]
First of all it is great lib...
I have couple of questions
1. Seems like transpose feature is not working i have tried data-transposition-pitches with different values but both sound and svgs showing same render. I am using guitar pro files.
2. Is there anyway to convert guitar pro files to alphatext on the fly?
3. When i resize the browser while it is playing alphaTab freezes 5-10 seconds with weird sound..how can i prevent?

**Comment by Danielku15** [comment, marked as answer]
1. Can you please file a full [bugreport](https://github.com/CoderLine/alphaTab/issues/new/choose)for this? I will need more info than "it is not working" to help you out here. 
2. Not yet. Exporters are on the back of the roadmap (1.5) https://github.com/orgs/CoderLine/projects. For 1.2 I plan to add a GuitarPro7 exporter. It is currently in development: https://github.com/CoderLine/alphaTab/pull/447
3.  This is quite a general request which must be special to your environment and setup. Due to poor standardization and terrible deicions of the Web Audio API (inparticular the AudioWorklets) alphaTab is still still doing some main audio playback logic on the UI thread. There can be many factors on a page which makes the browser slow and a slow browser causes the freezes and strange sounds. Root causes can be
* simply a bad performing browser (poor machine with too less CPU/GPU power)
* to complex website layout which causes additional load for the browser beside alphaTab operations. 
* other elements/scripts on your page causing additional load

To find out the root cause you will have to do some [performance profiling](https://developers.google.com/web/tools/chrome-devtools/rendering-tools) on your machine.
